src/plugins/shared_ux/public/components/toolbar/solution_toolbar/button/primary.stories.tsx

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React from 'react';
+import { SolutionToolbarButton } from './primary';
+import mdx from './primary.mdx';
+
+export default {
+  title: 'Solution Toolbar Button',
+  description: 'A button that is a part of the solution toolbar.',
+  parameters: {
+    docs: {
+      page: mdx,
+    },
+  },
+};
+
+export const Component = () => {
+  return <SolutionToolbarButton label={'Primary Action'} iconType="apps" />;
```

Comment:
Can we add a control with a few different icon types? Not a big deal if not, just curious.

Reply:
Smart! Added some more options to the control in [e2c59b3](https://github.com/elastic/kibana/pull/125998/commits/e2c59b3d528eb48fbaf7241f3b8576ad759c0fcd)